tests/unit/tests_high_five_base.cpp

```diff
@@ -410,7 +410,7 @@ BOOST_AUTO_TEST_CASE(HighFiveRefCountMove) {
         DataSet d1 = file.createDataSet(GROUP_NAME1 + DATASET_NAME, dataspace,
                                         AtomicType<double>());
 
-        double values[10][10] = {0};
+        double values[10][10] = {{0}};
```

Comment:
I've fixed the compiler warning here, but since you also fix it in #128 , I can remove this one here if you'd rather not rebase.

Reply:
I don't mind rebasing, I do it all the time. In any case, in any case it doesn't conflict